springwolf-plugins/springwolf-kafka-plugin/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/annotation/KafkaAsyncOperationBinding.java

```diff
@@ -26,14 +26,22 @@
     @Target({})
     @interface KafkaAsyncMessageBinding {
 
-        KafkaKeyTypes keyType() default KafkaKeyTypes.NO_KEY;
-
-        String description() default "";
+        KafkaAsyncKey key() default @KafkaAsyncKey(type = KafkaAsyncKey.KafkaKeyTypes.UNDEFINED_KEY);
```

Comment:
This should hide the mechanism that there is a value to mark that there was no configuration. If a user does not add the key annotation it will be UNDEFINED, if he adds the annotation, it will be automatically of String type without the need to explicitly specify it ...